packages/polling-controller/src/PollingController.test.ts

```diff
@@ -211,6 +211,26 @@ describe('PollingController', () => {
       await Promise.resolve();
       expect(controller.executePoll).toHaveBeenCalledTimes(2);
     });
+    it('should not poll twice for the same networkClientId and options', async () => {
```

Comment:
Maybe could we add a test where 2 different polls are started with the same networkClientId but different addresses (or other options) and show that they are running independently and can be disconnected independently to test that the key serialization is working as expected?